### ISSUE TITLE
UI/UX Landing Page

### ISSUE THREAD

**Issue by haifahrul** [haifahrul/code-converter#1]

**Comment by Everkers** [comment]
Hey @haifahrul, i would love to contribute to this one 😃

**Comment by haifahrul** [comment]
sure, go ahead 👍 😃.

The main concept is that the user chooses a converter tool like json to byte.
The left side is the code input and the right side is the conversion result

What do you think? 😄

**Comment by SankalpaFernando** [comment]
@haifahrul  This is a node project.isn't it?

**Comment by haifahrul** [comment]
Yup, I haven't had time to add `nodemon` and others tools or dependencies for develop.

My pleasure, if you have much time to setup it 😄 

> @haifahrul This is a node project.isn't it?

**Comment by Everkers** [comment]
> sure, go ahead 👍 😃.
> 
> The main concept is that the user chooses a converter tool like json to byte.
> The left side is the code input and the right side is the conversion result
> 
> What do you think? 😄

Sounds good, it would be great if we designed some simple wireframes and discuss it first?

**Comment by haifahrul** [comment]
@Everkers it's was created by @SankalpaFernando , so you can improve it if you want or open new issue
